components/src/atoms/StyledText/LegacyStyledText.tsx

```diff
@@ -58,7 +58,11 @@ const styleMap: { [tag: string]: FlattenSimpleInterpolation } = {
       ${TYPOGRAPHY.level3HeaderSemiBold}
     }
   `,
-  h4SemiBold: TYPOGRAPHY.level4HeaderSemiBold,
+  h4SemiBold: css`
+    @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
```

Comment:
The tests are failing because we need to include something like this 
```suggestion
    ${TYPOGRAPHY.level4HeaderSemiBold}
    @media ${RESPONSIVENESS.touchscreenMediaQuerySpecs} {
```